src/platform/monitoring/DowntimeNotification.jsx

```diff
@@ -52,18 +59,56 @@ function DowntimeNotificationWrapper({ status, children }) {
   return <div className="downtime-notification row-padded" data-status={status}>{children}</div>;
 }
 
+/**
+ * Downtime object definition
+ * @typedef {Object} Downtime
+ * @property {string} service
+ * @property {string} description
+ * @property {Date} startTime
+ * @property {Date} [endTime]
+ */
+
+/**
+ * React component used to conditionally render children components based on the status (down, down-approaching, or ok) of Vets.gov services.
+ * @module platorm/monitoring/DowntimeNotification
+ */
 class DowntimeNotification extends React.Component {
 
   static propTypes = {
+
+    /** The name of the consuming application, which will be displayed in downtime messaging. */
```

Comment:
I think I would rather see these comments in the `@property` tags above, so they show up whenever we start generating docs from JSDoc comments.